backend/tests/Util/DataLoaderTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace test\Util;
+
+use App\Domain\ValueObject\Requirements;
+use App\Util\DataLoader\ApiProviders;
+use App\Util\DataLoader\RequestedOutputType;
+use App\Util\DataLoader\RequestType;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use App\Repository\Repository;
+use App\Util\DataLoader\DataLoader;
+use test\Util\Fake\HttpClientFake;
+
+final class DataLoaderTest extends KernelTestCase
+{
+    /** @var DataLoader|null $dataLoader */
+    private $dataLoader;
+
+    private $projectDir;
+
+    private $expectedValuesPath;
+
+    protected function setUp()
+    {
+        $kernel = self::bootKernel();
+        /** @var Repository $repository */
+        $repository = $kernel->getContainer()->get('App\Repository\Repository');
+        $httpClientFake = new HttpClientFake();
+        $this->projectDir = $kernel->getProjectDir();
+        $this->expectedValuesPath = $kernel->getProjectDir() . '/tests/Util/ExpectedValues/';
+        $httpClientFake->setPath($kernel->getProjectDir());
+        $this->dataLoader = new DataLoader($repository, $httpClientFake);
+    }
+
+    private function clearDb(): void {
+        $storagePath =  $this->projectDir . '/storage/api/';
+        shell_exec('cd ' . $storagePath . ' && find . -name "*.json" -type f -delete');
```

Comment:
array_map('unlink', array_filter((array)glob($this->getParameter("images_host").'/*')));

w poprzednim projekcie jakoś tak to robiłem, testy nie powinny być zależne od platformy na której są wykonywane